experimental/dds/tree2/src/feature-libraries/chunked-forest/codec/format.ts

```diff
@@ -14,6 +16,32 @@ import {
 
 export const version = "unstable-development";
 
+// Compatible versions used for format/version validation.
+// If a new version is created, be sure to remove incompatible versions from this set.
+const validVersions = new Set([version]);
+
+const Versioned = Type.Object({
+	version: Type.String(),
```

Comment:
I believe the schema index example uses strings as well, but I think numbered versions are probably sufficient

Reply:
Changed version type to `Type.Number()`